Sources/MySQL/Bind.swift

```diff
@@ -18,7 +22,7 @@ public final class Bind {
     public typealias CBind = MYSQL_BIND
 
     /// MySQL represents Characters using signed integers.
```

Comment:
Agreed. After further inspection and testing, I think we should be safe. `Char` is only used while populating the buffer, and MySQL has no concept of "signed" vs "unsigned" - it's just raw bytes in a buffer. So in reality `UInt8` accounts for everything `Int8` was handling, but it also accounts for the characters that were causing overflow.